src/modules/shared/services/translation-service/translation.service.ts

```diff
@@ -9,4 +15,19 @@ export class TranslationService {
 
 		return response ?? {};
 	}
+
+	public static getLocale(): Locale {
+		if (isServerSideRendering()) {
+			return Locale.nl; // Window not available in server side rendering
+		}

```

Comment:
To my understanding this will make only the Dutch-translated version indexable by crawlers, am I missing something?